packages/components/src/TreeView/TreeView.test.js

```diff
@@ -36,8 +37,38 @@ const defaultProps = {
 };
 
 describe('TreeView', () => {
+	it('should be exported', () => {
+		expect(TreeView).toBe(ExportedTreeView);
+	});
 	it('should render normally with all buttons and custom labels', () => {
 		const wrapper = shallow(<TreeView {...defaultProps} />);
 		expect(wrapper.getNode()).toMatchSnapshot();
 	});
+	it('should call add action', () => {
```

Comment:
should be rephrased example 'user click on XXX should call `function/props` on XXX'